.travis.yml

```diff
@@ -101,6 +101,13 @@ script:
   - |
     if [[ -v HIVE_TESTS ]]; then
       presto-hive-hadoop2/bin/run_hive_tests.sh
+      if [[ -v S3_TESTS_AWS_ACCESS_KEY_ID && -v S3_TESTS_AWS_SECRET_ACCESS_KEY && -v S3_TESTS_BUCKET_ENDPOINT && -v S3_TESTS_BUCKET ]]; then
+        AWS_ACCESS_KEY_ID=$S3_TESTS_AWS_ACCESS_KEY_ID \
```

Comment:
start this (multi)line with `env` to highlight the fact you're _not_ setting anything permanently in the shell context. And indent the 2+ lines.